.github/workflows/windows-release.yml

```
@@ -0,0 +1,43 @@
+name: Publish Windows Releases
+on:
+  pull_request: null
+  release:
+    types: [published]
+
+permissions:
+  contents: write
+
+jobs:
+  get-extension-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.extension-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Get the extension matrix
+        id: extension-matrix
+        uses: php/php-windows-builder/extension-matrix@v1
+        with:
+          php-version-list: '8.0, 8.1, 8.2, 8.3, 8.4'
+          arch-list: 'x64, x86'
+          ts-list: 'ts'
+  build:
+    needs: get-extension-matrix
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix: ${{fromJson(needs.get-extension-matrix.outputs.matrix)}}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Fetch dependencies
+        run: |
+          curl -LO https://downloads.php.net/~windows/pecl/deps/pthreads-3.0.0-vs16-${{matrix.arch}}.zip
+          7z x pthreads-3.0.0-vs16-${{matrix.arch}}.zip -- -o..\deps
```

There is a spurious `--` which needs to be removed:
```suggestion
          7z x pthreads-3.0.0-vs16-${{matrix.arch}}.zip o..\deps
```
However, it would be better if php-windows-builder installs the dependency (see comment below).